vector-im/element-android
# [Device Manager] Unverified and inactive sessions list (PSG-698, PSG-696)

## Type of change


## Content
From the main session management screen, clicking view all button of Security Recommendations section navigates to standalone session list screen with the correct filter applied by default.


## Motivation and context
Fixes #7170.



## Screenshots / GIFs



|Unverified sessions|Inactive sessions|
|-|-|
|![unverified_sessions](https://user-images.githubusercontent.com/1254401/191033080-a820e5d7-36e0-40b8-a5c7-241e6408c30f.png)|![inactive_sessions](https://user-images.githubusercontent.com/1254401/191033151-1bbed9ba-b341-4b1d-9300-27510ce96163.png)|

## Tests



- Enable VectorFeatures.isNewDeviceManagementEnabled
- Navigate to Settings > Security & Privacy > Show All Sessions (V2, WIP)
- Click on View All button under the Security Recommendations section
- See the list of all sessions with the correct filter applied by default

## Tested devices

- OS version(s):

## Checklist
